tools/dynamatic/dynamatic.cpp

```diff
@@ -560,7 +562,20 @@ CommandResult Compile::execute(CommandArguments &args) {
     return CommandResult::FAIL;
 
   std::string script = state.getScriptsPath() + getSeparator() + "compile.sh";
-  std::string buffers = args.flags.contains(SIMPLE_BUFFERS) ? "1" : "0";
+  std::string buffers = "simple-buffers";
```

Comment:
Isn't this a good time to rename the not-very-meaningful "simple-buffers" with "on-merges", which is both more expressive and the name of the associated algorithm in the buffer placement pass?